scss/_base/_forms.scss

```diff
@@ -1,59 +1,99 @@
+// A label and a form element.
+//
+// .-padded - Adds extra padding above and below field group.
+// .-inline - Form elements appear side-by-side.
+//
+// Styleguide Forms - Field Groups
+.field-group {
+  margin-bottom: ($base-spacing / 2);
```

Comment:
Should this follow suit w/ other spaced elements, like `<p>`? Something like:

`````
.field-group + .field-group {
  margin-top: ($base-spacing / 2);
}
`````